beta/next-env.d.ts

```diff
@@ -1,5 +1,4 @@
 /// <reference types="next" />
-/// <reference types="next/types/global" />
```

Comment:
Seems like you have deleted this line by mistake. Can you revert this.

Reply:
done